doc/src/fix_ave_moments.rst

```diff
@@ -0,0 +1,293 @@
+
+For *kurtosis*, the adjusted Fisher--Pearson standardized moment
+:math:`G_2 = \frac{k_4}{k_2^2}` is calculated.
+
+----------
+
+Fix invocation and output can be modified by optional keywords.
+
+The *start* keyword specifies that the first computation should be no
+earlier than the step number given (but will still occur on a multiple
+of *Nfreq*).  The default is step 0.  Often input values can be 0.0 at
+time 0, so setting *start* to a larger value can avoid including a 0.0
+in a longer series.
+
+The *history* keyword stores the Nrecent most recent outputs on Nfreq
+timesteps, so they can be accessed as global outputs of the fix.  By
+default, only the most recent output is accessible. For example, if
+history 10 is specified and Nfreq = 1000, then on timestep 20000, the
+Nfreq outputs from steps 20000, 19000, ... 11000 are available for
+access.  See below for details on how to access the history values.
+
+For example, this will store the outputs of the previous 10 Nfreq
+time steps, i.e. a window of 10000 time steps:
+
+.. code-block:: LAMMPS
+
+   fix 1 all ave/moments 1 200 1000 v_volume mean history 10
+
+The previous results can be accessed as values in a global array output
+by this fix. Each column of the array is the vector output of the N-th
+preceding Nfreq timestep.  For example, the most recent result of the
+third input value would be accessed as "f_name[3][1]", "f_name[3][4]"
+is the 4th most recent and so on.  The current vector output is always
+the first column of the array, corresponding to the most recent result.
+
+To illustrate the utility of keeping output history, consider using
+this fix in conjunction with :doc:`fix halt <fix_halt>` to stop a run
+automatically if a quantity is converged to within some desired tolerance:
+
+.. code-block:: LAMMPS
+
+   variable target equal etot
+   fix aveg all ave/moments 1 200 1000 v_target mean stddev history 10
+   variable stopcond equal "abs(f_aveg[1]-f_aveg[1][10])<f_aveg[2]"
+   fix fhalt all halt 1000 v_stopcond == 1
+
+In this example, every 1000 time steps, the average and standard
+deviation of the total energy over the previous 200 time steps are
+calculated.  If the difference between the most recent and 10-th most
+recent average is lower than the most recent standard deviation, the run
+is stopped.
+
+----------
+
+Restart, fix_modify, output, run start/stop, minimize info
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+No information about this fix is written to :doc:`binary restart files
+<restart>`.
+
+This fix produces a global vector and global array which can be accessed
+by various :doc:`output commands <Howto_output>`.  The values can be
+accessed on any time step, but may not be current.
+
+A global vector is produced with the # of elements = number of moments *
+number of inputs.  The moments are output in the order given in the fix
+definition.  An array is produced having # of rows = same as vector output,
```

Comment:
... # of rows = length of vector output (with an ordering which matches the vector), and # of columns = ...